src/services/membership/membership.dto.result.ecoverse.entry.ts

```diff
@@ -0,0 +1,27 @@
+import { Field, ObjectType } from '@nestjs/graphql';
+import { MembershipResultEntry } from './membership.dto.result.entry';
+
+@ObjectType()
+export class MembershipEcoverseResultEntry {
+  @Field(() => String, {
+    nullable: true,
+    description: 'The name of the Ecoverse.',
+  })
+  name = '';
+
+  @Field(() => String, {
```

Comment:
The bulk of the IDs we have (excluding TextIDs) are @Field(() => ID) mapped to number in JS.

Reply:
left this as generic string for now as need to pick up IDs systematically with global ids epic...